docs/modules/ROOT/pages/documentation/ecosystem/maven-bom.adoc

```diff
@@ -26,7 +26,7 @@ Add following snippet to your Maven project:
         <dependency>
             <groupId>fish.payara.api</groupId>
             <artifactId>payara-bom</artifactId>
-            <version>5.194</version>
+            <version>5.2020.2</version>
```

Comment:
Can we configure this based on the `currentVersion` Antora property?

Reply:
No, I tried. Antora doesn't perform variable substitution in code blocks.